cmd/servicelog/list.go

```diff
@@ -34,6 +34,10 @@ var listCmd = &cobra.Command{
 			}
 		}()
 
+		if len(args) != 1 {
+			log.Infof("Too many arguments. Expected 1 got %d", len(args))
```

Comment:
@georgettica  do we want to exit when the args don't match or is this not necessary?

Reply:
No, I accepted the comment but meant for this to be informational, in case we hit this later on